resources/js/components/App.tsx

```diff
@@ -1,10 +1,16 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+
+import Login from "./Login";
+import Signup from "./Signup";
+
 function App() {
     return (
-        <div className="app">
-            <h1>Hi, my name is Brix!</h1>
-            <p>I am currently building an E-Learning System App.</p>
-            <p>Please standby. It is still in progress...</p>
-        </div>
+        <BrowserRouter>
+            <Routes>
```

Comment:
Use a different file for the routes and import here instead.

Reply:
Refactored it to have a single View component in the App